microsoft-edge/webview2/concepts/developer-guide.md

```diff
@@ -6,79 +6,91 @@ ms.author: msedgedevrel
+We recommend using the Evergreen WebView2 Runtime for most WebView2 apps, rather than using the Fixed Version runtime.
 
-If you use the Evergreen runtime, before running your WebView2 app, test whether the Evergreen WebView2 Runtime is installed on the client.  See [Deploying the Evergreen WebView2 Runtime](../concepts/distribution.md#deploying-the-evergreen-webview2-runtime).
+The Evergreen runtime updates automatically on the client, so that the latest features and security patches are available to your WebView2 app.  The Evergreen runtime also requires less storage space on the disk than the Fixed Version runtime.  Fixed Version runtime distribution is only recommended for apps that have strict compatibility requirements.
 
-<!-- ====================================================================== -->
-## Run compatibility tests regularly when using the Evergreen Runtime
+For more information about the benefits of the Evergreen Runtime, see [Details about the Evergreen Runtime distribution mode](../concepts/distribution.md#details-about-the-evergreen-runtime-distribution-mode) in _Distribute your app and the WebView2 Runtime_.  
 
-When using the Evergreen WebView2 Runtime, the runtime updates automatically, so you must regularly run compatibility tests.  To ensure that your WebView2 app will continue to work as expected, test your web content in the WebView2 control against the Microsoft Edge preview channels (Beta, Dev, or Canary).  The  preview channels are also called _Insider channels_.
+To help ensure that your WebView2 app works well with the Evergreen WebView2 Runtime, use the recommended practices in the subsections below:
+* [Check to make sure the WebView2 Runtime is installed](#check-to-make-sure-the-webview2-runtime-is-installed).
+* [Handle updates of the WebView2 Runtime](#handle-updates-of-the-webview2-runtime).
+* [Do forward-compatibility testing using Edge preview channels](#do-forward-compatibility-testing-using-edge-preview-channels).
+* [Use feature detection for newer APIs](#use-feature-detection-for-newer-apis).
 
-The above guidance is similar to the guidance for web developers.  To test your app for forward-compatibility, see [Prerelease testing using preview channels](../how-to/prerelease-testing.md) and [Self-host by deploying preview channels](../how-to/self-hosting.md).
 
-To download the preview channels of Microsoft Edge, go to [Become a Microsoft Edge Insider](https://www.microsoft.com/edge/download/insider).
+<!-- ------------------------------ -->
+#### Check to make sure the WebView2 Runtime is installed
 
+When using the Evergreen WebView2 Runtime, before your app uses the WebView2 control, programmatically check to make sure that the WebView2 Runtime is installed on the client.
```

Comment:
```suggestion
When using the Evergreen WebView2 Runtime, before your app uses the WebView2 control, programmatically check to make sure that the Evergreen WebView2 Runtime is installed on the client.
```
check specifically for the "Evergreen" runtime?  Does the linked section say how to check for Evergreen vs Fixed Version?  I think the linked section just says how to check yes/no is Runtime installed or not.  Earlier draft said "Evergreen" but I suspect that's too specific and mis-emphasis.
@victorhuangwq @david-risney

Reply:
rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/developer-guide?branch=pr-en-us-3195#check-to-make-sure-the-webview2-runtime-is-installed
it links to section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution?branch=pr-en-us-3195&tabs=dotnetcsharp#detect-if-a-suitable-webview2-runtime-is-already-installed

Reply:
removed suitable